components/org.wso2.carbon.identity.oidc.session/src/test/java/org/wso2/carbon/identity/oidc/session/cache/OIDCSessionDataCacheTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oidc.session.cache;
+
+import org.testng.annotations.Test;
+import org.wso2.carbon.base.MultitenantConstants;
+import org.wso2.carbon.identity.oidc.session.servlet.TestUtil;
+
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+
+/**
+ * Unit test coverage for OIDCSessionDataCache class.
+ */
+public class OIDCSessionDataCacheTest {
+
+    @Test
+    public void testGetInstance() {
+
+        assertNotNull(OIDCSessionDataCache.getInstance(), "OIDCSessionDataCache is null.");
+    }
+
+    @Test
+    public void testAddToCache() {
```

Comment:
Can we add a couple of test cases to cover the scenario where a cache miss happens at the in memory cache and the data is fetched from the session data store?

Reply:
Fixed by 05697c21b56abf3d790d8577ef4e072b03f2922a